test/index.test.js

```diff
@@ -1053,6 +1053,50 @@ jobs:
                 });
         });
 
+        it('promises to get empty content when file is not found', () => {
+            const err = new Error('githubError');
+
+            err.status = 404;
+
+            githubMock.repos.getContents.rejects(err);
+
+            return scm.getFile(config)
+                .then((data) => {
+                    assert.deepEqual(data, '');
+
+                    assert.calledWith(githubMock.repos.getContents, {
+                        owner: 'screwdriver-cd',
+                        repo: 'models',
+                        path: config.path,
+                        ref: config.ref
+                    });
+                });
+        });
+
+        it('returns an error when github command fails', () => {
```

Comment:
Is there the same test case on L1120?

Reply:
Thx. I deleted this case.
I think I pasted it in by mistake.